### ISSUE TITLE
`averageBlockTime` should not be zero

### ISSUE THREAD

**Issue by magnetto90** [overlay-market/v1-core#132]
In Arbitrum the average block time is ≈0.25s but not 0. [🔗](https://arbiscan.io/chart/blocktime)

This functions divides by `averageBlockTime`.

```solidity
    function backRunBound(Oracle.Data memory data) public view returns (uint256) {
        uint256 averageBlockTime = params.get(Risk.Parameters.AverageBlockTime);
        uint256 window = (data.macroWindow * ONE) / averageBlockTime;
        uint256 delta = params.get(Risk.Parameters.Delta);
        return delta.mulDown(data.reserveOverMicroWindow).mulDown(window).mulDown(2 * ONE);
    }
 ```
 
The only line where this functions is called is here.

```solidity
    function capNotionalAdjustedForBounds(Oracle.Data memory data, uint256 cap)
        public
        view
        returns (uint256)
    {
        if (data.hasReserve) {
            // Adjust cap downward if exceeds bounds from front run attack
            cap = Math.min(cap, frontRunBound(data));

            // Adjust cap downward if exceeds bounds from back run attack
            cap = Math.min(cap, backRunBound(data));
        }
        return cap;
    }
```
    
🛑 So, the problem should arise when the averageBlockTime == 0 && data.hasReserve == true 

Note: `averageBlockTime` limit is configured [here](https://github.com/overlay-market/v1-core/blob/a5a12f856fc3434c4450aaa61fc3c4f27b56b14c/contracts/OverlayV1Factory.sol#L34).

**Comment by magnetto90** [comment]
#133 modifies minimum `averageBlockTime `

**Comment by magnetto90** [comment]
Fixed with #133
